src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -109,7 +119,7 @@ export class EditorGroupModel extends Disposable {
 	private readonly _onDidChangeEditorCapabilities = this._register(new Emitter<EditorInput>());
 	readonly onDidChangeEditorCapabilities = this._onDidChangeEditorCapabilities.event;
 
-	private readonly _onDidMoveEditor = this._register(new Emitter<EditorInput>());
+	private readonly _onDidMoveEditor = this._register(new Emitter<EditorMoveEvent>());
```

Comment:
A test for the correct properties of this new event would be nice: `editorGroupModel.test.ts`

Reply:
Added